lib/internal/util/inspector.js

```diff
@@ -20,6 +18,49 @@ function sendInspectorCommand(cb, onError) {
   }
 }
 
+// Create a special require function for the inspector command line API
+function installConsoleExtensions(commandLineApi) {
+  if (commandLineApi.require) { return; }
+  const { tryGetCwd } = require('internal/process/execution');
+  const CJSModule = require('internal/modules/cjs/loader');
+  const { makeRequireFunction } = require('internal/modules/cjs/helpers');
+  const consoleAPIModule = new CJSModule('<inspector console>');
+  const cwd = tryGetCwd();
+  consoleAPIModule.paths =
+      CJSModule._nodeModulePaths(cwd).concat(CJSModule.globalPaths);
+  commandLineApi.require = makeRequireFunction(consoleAPIModule);
+}
```

Comment:
👆 Could someone explain this bit. I'm unfamiliar with the `'<inspector console>'` and `commandLineApi.require` bits. Is there a `require` exposed in the Chrome inspector console?

Reply:
The inspector console is the one in the chrome inspector. By doing this, a `require` is exposed when you connect a chrome inspector to node.

Reply:
Is this something user accessible?

Reply:
@jdalton: it's accessed when using `require` in DevTools console.